test/x86/sse4.2.c

```diff
@@ -24,9 +24,231 @@
 #define SIMDE_TESTS_CURRENT_ISAX sse4_2
 #include <test/x86/test-x86-internal.h>
 #include <simde/x86/sse4.2.h>
+#include <assert.h>
 
 #if defined(SIMDE_X86_SSE4_2_NATIVE) || defined(SIMDE_NO_NATIVE) || defined(SIMDE_ALWAYS_BUILD_NATIVE_TESTS)
 
+#if defined(SIMDE_X86_SSE4_2_NATIVE)
+#define SIMDE_SIDD_UBYTE_OPS _SIDD_UBYTE_OPS
```

Comment:
Small style issue.  The inside of the #if block should be indented; i.e., 

```c
#if defined(SIMDE_X86_SSE4_2_NATIVE)
  #define SIMDE_SIDD_UBYTE_OPS _SIDD_UBYTE_OPS
```

This style is a little weird, most projects would prefer something like

```c
#if defined(SIMDE_X86_SSE4_2_NATIVE)
#  define SIMDE_SIDD_UBYTE_OPS _SIDD_UBYTE_OPS
```

But for SIMDe I'm trying to use the former since it's a bit easier to handle consistently when automatically generating code.  Basically, preprocessor conditionals are indented just like regular conditionals; they start at the same indentation as normal code, and all code inside of them (regardless of whether we're talking about normal code or preprocessor code) gets indented.